docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/ecommerce/index.md

```diff
@@ -65,6 +65,22 @@ All variables in Snowplow packages start with `snowplow__` but we have removed t
 | -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
 | `databricks_catalog` | The catalogue your atomic events table is in. Depending on the use case it should either be the catalog (for Unity Catalog users from databricks connector 1.1.1 onwards, defaulted to `hive_metastore`) or the same value as your `snowplow__atomic_schema` (unless changed it should be 'atomic'). |         |
 
+</TabItem>
+<TabItem value="redshift+postgres" label="Redshift & Postgres">
+
+Redshift and Postgres use a [shredded](/docs/destinations/warehouses-and-lakes/rdb/transforming-enriched-data/#shredded-data) approach for the context tables, so these variables are used to identify where they are, if different from the expected schema and table name. They must be passed in a stringified `source` function as the defaults below show.
```

Comment:
Can a “stringified `source` function” be something other than a single-quoted table name? If not, I think calling it a single-quoted table name is a bit friendlier.

Reply:
Bad C+P on my part, this packages actually has a different format for these variables than the rest.